docs/examples/TimePlotterExample.py

```diff
@@ -0,0 +1,147 @@
+from stonesoup.types.state import GaussianState
+from stonesoup.updater.kalman import KalmanUpdater
+
+# %%
+# Set up the platform and detection simulators
+
+# Models
+transition_model = CombinedLinearGaussianTransitionModel(
+    [ConstantVelocity(1), ConstantVelocity(1)])
+measurement_model = LinearGaussian(4, [0, 2], np.diag([0.5, 0.5]))
+
+start_time = datetime.datetime.now()
+
+
+# Simulators
+groundtruth_sim = MultiTargetGroundTruthSimulator(
+    transition_model=transition_model,
+    initial_state=GaussianState(
+        StateVector([[0], [0], [0], [0]]),
+        CovarianceMatrix(np.diag([1000, 10, 1000, 10])),
+        timestamp=start_time),
+    timestep=datetime.timedelta(seconds=5),
+    number_steps=60,
+    birth_rate=0.2,
+    death_probability=0.05
+)
+detection_sim = SimpleDetectionSimulator(
+    groundtruth=groundtruth_sim,
+    measurement_model=measurement_model,
+    meas_range=np.array([[-1, 1], [-1, 1]]) * 5000,  # Area to generate clutter
+    detection_probability=0.9,
+    clutter_rate=1,
+)
+
+# %%
+# Set up the tracker
+
+# Filter
+predictor = KalmanPredictor(transition_model)
+updater = KalmanUpdater(measurement_model)
+
+# Data Associator
+hypothesiser = DistanceHypothesiser(predictor, updater, Mahalanobis(), missed_distance=3)
+data_associator = GNNWith2DAssignment(hypothesiser)
+
+# Initiator & Deleter
+deleter = CovarianceBasedDeleter(covar_trace_thresh=1E2)
+initiator = MultiMeasurementInitiator(
+    GaussianState(np.array([[0], [0], [0], [0]]),
+                  np.diag([0, 100, 0, 1000]),
+                  timestamp=start_time),
+    measurement_model=measurement_model,
+    deleter=deleter,
+    data_associator=data_associator,
+    updater=updater,
+    min_points=2,
+)
+
+# Tracker
+tracker = MultiTargetTracker(
+    initiator=initiator,
+    deleter=deleter,
+    detector=detection_sim,
+    data_associator=data_associator,
+    updater=updater,
+)
+
+# %%
+# Run the simulation
+
+groundtruth = set()
+detections = set()
+all_tracks = set()
+
+for time, tracks in tracker:
+    groundtruth.update(groundtruth_sim.groundtruth_paths)
+    detections.update(detection_sim.detections)
+    all_tracks.update(tracks)
+
+
+# %%
+# Create the Animation
+# --------------------
+from stonesoup.plotter import AnimationPlotter
+from matplotlib import pyplot as plt
+
+
+# %%
+# Create the plotter object and use the plot_# functions to assign the data to plot
+plotter = AnimationPlotter()
+plotter.plot_ground_truths(groundtruth, mapping=[0, 2])
+plotter.plot_measurements(detections, mapping=[0, 2])
+plotter.plot_tracks(all_tracks, mapping=[0, 2])
+
+# %%
+# Run the Animation
+# -----------------
+# The times to refresh the animation are chosen to co-inside with the simulator times
+#
+# To avoid the figure becoming too cluttered with old information, states older than 60 seconds will
+# not be shown. The figure isn't animated inside a sphinx gallery but does work when ran from the
+# command line
```

Comment:
I've added a line below so the animation would load in a Jupyter notebook version but would it be possible to include a pre-saved video in the sphinx gallery version? @sdhiscocks is there a way to do this at all?

Reply:
The latest commit shows the animation in Sphinx